mlir/include/mlir/Dialect/Rock/IR/ConvolutionDims.h

```diff
@@ -23,20 +25,19 @@ namespace rock {
 
 /// Structure for holding the dimensions of a convolution problem
 struct ConvolutionDims {
-  int64_t y;
-  int64_t x;
-  int64_t ho;
-  int64_t wo;
-  int64_t hi;
-  int64_t wi;
+  SmallVector<int64_t, 5> fil;
```

Comment:
These guys become arrays, which affects both how ConvolutionDims is constructed by its users and how the data is extracted and passed on from the struct.